Tests/Unit/EmogrifierTest.php

```diff
@@ -1997,10 +1997,7 @@ public function emogrifierMapsSuitableCssToHtmlIfFeatureIsEnabled($body, $css, $
         $this->subject->enableCssToHtmlMapping();
         $html = $this->subject->emogrify();
 
-        self::assertContains(
-            '<' . $tagName . ' ' . $attributes,
-            $html
-        );
+        self::assertRegExp('/<' . $tagName . '[^>]+' . $attributes . '/', $html);
```

Comment:
You'll need to use preg_quote on $tagName (just to be consistent) and $attributes as you are now converting this matcher to a regular expression.